renderdoc/driver/dxgi/dxgi_common.cpp

```diff
@@ -1449,6 +1449,8 @@ DXGI_FORMAT MakeDXGIFormat(ResourceFormat fmt)
   {
     if(fmt.compByteWidth == 4)
       ret = DXGI_FORMAT_R32_TYPELESS;
+    else if(fmt.compByteWidth == 3)
+      ret = DXGI_FORMAT_R24_UNORM_X8_TYPELESS;
```

Comment:
In my case, the texture size allocated on the GLReplay side is 4 bytes per pixel, and since it's calling the emulated version of _glGetTexImage with format = eGL_DEPTH_COMPONENT, the operation bailed out without actually getting the image back.
I guess it's better to implement D24 for real, but since I don't have a use case yet, I'll leave this one out until I do.